src/ui/components/TestSuite/views/TestRecording/TestRecordingEvents/UserActionEventRow.tsx

```diff
@@ -171,12 +176,14 @@ export default memo(function UserActionEventRow({
         </span>
       </div>
       {showBadge && (
-        <Suspense fallback={<Loader />}>
-          <Badge isSelected={isSelected} timeStampedPoint={resultTimeStampedPoint} />
-        </Suspense>
+        <div className="flex-none">
+          <Suspense fallback={<Loader />}>
+            <Badge isSelected={isSelected} timeStampedPoint={resultTimeStampedPoint} />
+          </Suspense>
+        </div>
       )}
       {showJumpToCode && jumpToCodeAnnotation && (
-        <div className={styles.JumpToCodeButton}>
+        <div className="flex-none">
```

Comment:
I'm surprised to see you remove a CSS modules class to use a tailwind class in this change. I don't have a strong opinion on which we're using but I'm curious the rationale.

Also, if we are removing usage of classes, let's confirm if we can remove the CSS too!

Reply:
Hm, let me double-check this. It's possible I did a dumb.

Reply:
Same comment as above. Let's stick with the CSS modules.